### ISSUE TITLE
Incorrect References to PostgreSQL

### ISSUE THREAD

**Issue by blakhani-msft** [Azure/azure-cli-extensions#3882]
I tried creating a Azure SQL Database using az sql up command. The creation went through fine. 
The only minor issue is that the output on the screen shows reference to PostgreSQL. Here is the output and highlighted section needs to be removed/corrected.

<snip>
If SQL server declines your IP address, please create a new firewall rule using:
    `az sql server firewall-rule create -g blakhani-azuresqldb1 -s blakhanisql1 -n {rule_name} --start-ip-address {ip_address} --end-ip-address {ip_address}`
Successfully Connected to PostgreSQL.
Ran Database Query: `CREATE USER root WITH PASSWORD = 'HS_via_azsqlup1'`
Ran Database Query: `GRANT ALL TO root`
{
  "connectionStrings": {
</snip>

**Command Name**
`az sql up
Extension Name: db-up. Version: 0.2.5.`

## To Reproduce:
Run the command and watch the output.

- `az sql up --resource-group {} --location {} --server-name {} --database-name {} --admin-user {} --admin-password {}`

## Expected Behavior
There is no issue with the behavior command,. Its just the output which needs correction. 

## Environment Summary
```
Linux-5.4.0-1058-azure-x86_64-with-debian-10.2 (Cloud Shell), Common Base Linux Delridge (quinault)
Python 3.6.10
Installer: DEB

azure-cli 2.28.0

Extensions:
db-up 0.2.5
ai-examples 0.2.5
ssh 0.1.6

```

**Comment by yonzhan** [comment]
sql up

**Comment by evelyn-ys** [comment]
Will be fixed in #3874
